pkg/model/serializer_test.go

```diff
@@ -1778,3 +1778,41 @@ compose:
 		})
 	}
 }
+
+func TestManifestBuildUnmarshalling(t *testing.T) {
+	tests := []struct {
+		name            string
+		buildManifest   []byte
+		expected        ManifestBuild
+		isErrorExpected bool
+	}{
+		{
+			name: "build params at manifest",
+			buildManifest: []byte(`firstBuild: ./first
+secondBuild:
```

Comment:
I would use a sample with a value for every field supported in the build section